modules/imgproc/src/pyramids.cpp

```diff
@@ -1266,6 +1266,8 @@ static bool ocl_pyrUp( InputArray _src, OutputArray _dst, const Size& _dsz, int
 
 }
 
+// TODO: enable it when the corresponding IPP code is fixed
+#if 0
```

Comment:
It's already disabled.
modules/core/include/opencv2/core/private.hpp:198:#define IPP_DISABLE_PYRAMIDS_DOWN       1 // Different results